tendrl/notifier/notification/handlers/mail_handler.py

```diff
@@ -58,9 +60,13 @@ def get_config_help(self):
         return config_help
 
     def set_destinations(self):
-        # TODO(anmolbabu):User configuration will not just be destination email
-        # but it would also include user level subscriptions the capability
-        # underneath needs to be enhanced for that.
+        # TODO(gowtham):Get user ids from indexes who enabled
+        # email notification
+        pass
+
+    def get_alert_destinations(self):
+        # TODO(gowtham): get user emailds only who have enabled
+        # email notfication
```

Comment:
after found user id or names who are all enabled email notification, i will pass the those details to this function to find email ids of that particular users. for now i am getting all user emails